docs/docs/authentication/index.mdx

```diff
@@ -5,7 +5,7 @@ sidebar_position: 1
 image: /img/og/authentication.png
 ---
 
-Nhost Authentication is a ready-to-use authentication service that is integrated with the [GraphQL API](/graphql) and its permission system from Hasura.
+Nhost Authentication is a ready-to-use authentication service that is integrated with the [GraphQL API](/graphql) and its permission system from Hasura. This allows you to easily add user authentication to your application without having to build and maintain your own authentication system.
```

Comment:
I wonder if “Nhost Authentication” should be called “Nhost Auth” here too. I saw you have changed it in another file.

Reply:
I think we're using "Authentication" more than "Auth" so I'll use "Authentication" everywhere for now. 

But I think we should consider using "Auth" instead. That we can do in a follow-up PR.